src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar primary-search-bar"
+      data-qa="big"
+    >
+      <input
+        class="search-bar-input search-bar-input--primary"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar secondary-search-bar"
+      data-qa="small"
+    >
+      <input
+        class="search-bar-input search-bar-input--secondary"
+        type="text"
+        data-qa="hover"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
   </body>
```

Comment:
In terms of HTML content, it would be better to use semantic tags where possible. For example, you could wrap the `form` elements in a `section` or `main` tag to indicate that they are the main content of the page. It will improve your page's SEO and help screen readers.